apps/src/templates/manageStudents/AddMultipleStudents.jsx

```diff
@@ -0,0 +1,83 @@
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {addMultipleAddRows} from './manageStudentsRedux';
+import Button from '../Button';
+import i18n from "@cdo/locale";
+import BaseDialog from '../BaseDialog';
+import DialogFooter from "../teacherDashboard/DialogFooter";
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  }
+};
+
+class AddMultipleStudents extends Component {
+  static propTypes = {
+    // Provided by redux
+    addMultipleStudents: PropTypes.func.isRequired,
+  };
+
+  state = {
+    openDialog: false,
```

Comment:
Super-picky nit: I'd name this something like `isDialogOpen`.  `openDialog` sounds like an action to me.  In fact, you  have a public method `openDialog` just below.